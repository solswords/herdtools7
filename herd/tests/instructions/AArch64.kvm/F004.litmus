AArch64 F004
{
0:X2=0;
}
  P0          | P0.F           ;
L0:           | MOV W2,#1      ;
  SVC #0      | ERET           ;
L1:           |                ;
  ADD W2,W2,#2|                ;

forall 0:X2=3;