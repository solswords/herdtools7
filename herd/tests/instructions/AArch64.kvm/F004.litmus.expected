Test F004 Required
States 1
0:X2=3;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X2=3)
Observation F004 Always 1 0
Hash=e06f7ca51d2491ed933049436a3069fd

